.github/ISSUE_TEMPLATE/roadmap-item.md

```diff
@@ -0,0 +1,28 @@
+---
+name: 'Roadmap Item'
+about: Create roadmap item for this project
+title: Roadmap Item =description=
+labels: kind/roadmap
+assignees: ''
+
+---
+
+# O3DE Roadmap Item Template
+
+### When using this template, you do not have to fill out every question below. The questions are there for guidance.
+
+This roadmap item template should be used for any feature that shows in the O3DE public roadmap. The issue communicates the initiative and vision of the SIG.
+
+# ----- DELETE EVERYTHING FROM THE TOP TO THE SUMMARY LINE BELOW WHEN USING TEMPLATE ----- #
```

Comment:
Nit: consider using input forms so you can provide guidance without cluttering up the issue.

See https://github.com/o3de/o3de/pull/13843 for example.

Reply:
Thanks for the suggestions. I'll put this as a backlog for future improvement!